### PR TITLE
Card component

### PR DESCRIPTION
**Pull request by Mussuranae**
Display a card

### CODE REVIEW

File: src/app/display/card/card.component.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<mat-card class="movie-card">
+    <img mat-card-image [src]="movie.poster" alt="Movie's poster">
+    <mat-card-content>
+      <mat-card-title-group>
+          <mat-card-title  class="mat-title">{{ movie.title.original }}</mat-card-title>
```
Comment (GrandSchtroumpf) [inline src/app/display/card/card.component.html]: Here this is a little bit special. As you already have a `mat-card-title` you don't have to specify the typography. Use typography on regular html text/heading tags, not the custom one.

File: src/app/display/card/card.component.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
+.movie-card {
```
Comment (GrandSchtroumpf) [inline src/app/display/card/card.component.scss]: Don't use classes to target an element. Do : `mat-card` directly.

File: src/app/display/card/card.component.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    box-sizing: border-box;
```
Comment (GrandSchtroumpf) [inline src/app/display/card/card.component.scss]: Never set box-sizing on an individual element.

File: src/app/display/card/card.component.scss [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    height: 393px;
```
Comment (GrandSchtroumpf) [inline src/app/display/card/card.component.scss]: Height should no be forced.

File: src/app/display/card/card.component.scss [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    width: 312px;
```
Comment (GrandSchtroumpf) [inline src/app/display/card/card.component.scss]: the width should be set at the parent level.

File: src/app/display/card/card.component.scss [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    border: 1px solid #292929;
```
Comment (GrandSchtroumpf) [inline src/app/display/card/card.component.scss]: NEVER change default style ;)

File: src/app/display/card/card.component.scss [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    border-radius: 10px;
+    background-color: #0E0D1B;
+    box-shadow: -10px 11px 32px 0 rgba(0,0,0,0.14);
+}
+.mat-card-image {
```
Comment (GrandSchtroumpf) [inline src/app/display/card/card.component.scss]: All the style below should be removed. Don't change default style.

File: src/app/pages/movie-list/movie-list.component.html [hunk 8]
```diff
@@ -1 +1,5 @@
-<p>movie-list works!</p>
+<div fxLayout="row">
```
Comment (GrandSchtroumpf) [inline src/app/pages/movie-list/movie-list.component.html]: Use section. Avoid div for top level container.

File: src/app/movie.MOCK.ts [hunk 9]
```diff
@@ -8,7 +8,31 @@ export const MOVIES = [
         international: 'Bonjour',
       },
       directors: ['Chan-uk Park'],
-      poster: '',
+      poster: 'https://m.media-amazon.com/images/M/MV5BMjU1MTA3MzA2MV5BMl5BanBnXkFtZTgwNTUxMjQ4NzM@._V1_UY1200_CR106,0,630,1200_AL_.jpg',
```
Comment (GrandSchtroumpf) [inline src/app/movie.MOCK.ts]: I think there'll be conflict with Dajung if you're using the same mock file for your components.

File: src/app/pages/movie-list/movie-list.component.html (hunk 8 above)
Comment (Mussuranae) [reply src/app/pages/movie-list/movie-list.component.html]: changed

File: src/app/display/card/card.component.scss [hunk 10]
```diff
@@ -0,0 +1,7 @@
+mat-card {
+    height: auto;
+    width: 312px;
+    border-radius: 10px;
```
Comment (GrandSchtroumpf) [inline src/app/display/card/card.component.scss]: Remove border, background and box shadow.

**Pull request merged**